main_routers/system_router.py

```
@@ -706,16 +911,36 @@ async def proxy_image(image_path: str):
-        from utils.web_scraper import fetch_trending_content, format_trending_content, fetch_window_context_content, format_window_context_content
+        from utils.web_scraper import (
+            fetch_trending_content, format_trending_content,
+            fetch_window_context_content, format_window_context_content,
+            fetch_video_content, format_video_content,
+            fetch_news_content, format_news_content
+        )
+        from config.prompts_sys import get_proactive_screen_prompt, get_proactive_generate_prompt, get_proactive_chat_rewrite_prompt
         
-        # 获取当前角色数据
-        master_name_current, her_name_current, _, _, _, _, _, _, _, _ = _config_manager.get_character_data()
+        # 获取当前角色数据（包括完整人设）
```

Ruff (0.15.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)